recipes-debian/AppArmor/apparmor_debian.bb

```diff
@@ -0,0 +1,204 @@
+    fi
+
+    # aa-easyprof is installed by python-tools-setup.py, fix it up
+    sed -i -e 's:/usr/bin/env.*:/usr/bin/python3:' ${D}${bindir}/aa-easyprof
+    chmod 0755 ${D}${bindir}/aa-easyprof
+
+    install ${WORKDIR}/apparmor ${D}/${INIT_D_DIR}/apparmor
+    install ${WORKDIR}/functions ${D}/lib/apparmor
+    install -d ${D}${systemd_system_unitdir}
+    install ${WORKDIR}/apparmor.service ${D}${systemd_system_unitdir}
+}
+
+do_install_append() {
+    #Install files add by debian package
+    #debian/apparmor.install
+    install -d ${D}/usr/share/apport/package-hooks
+    install ${S}/debian/apport/source_apparmor.py ${D}/usr/share/apport/package-hooks
+    install -d ${D}/usr/share/apparmor-features
+    install ${S}/debian/features ${D}/usr/share/apparmor-features
+    install ${S}/debian/lib/apparmor/profile-load ${D}/lib/apparmor
+    install ${S}/parser/apparmor.systemd ${D}/lib/apparmor
+    install ${S}/parser/rc.apparmor.functions ${D}/lib/apparmor/
+    install ${S}/parser/aa-teardown ${D}${sbindir}
+    install -D ${S}/debian/apparmor.lintian-overrides ${S}/usr/share/lintian/overrides/apparmor
+    #debian/apparmor-utils.install
+    install ${S}/debian/aa-update-browser ${D}${sbindir}
+    install -D ${S}/debian/vim-apparmor.yaml ${D}/usr/share/vim/registry
+    install -D ${D}/usr/share/apparmor/apparmor.vim ${D}/usr/share/vim/addons/syntax
+    install -D ${S}/debian/apparmor-utils.lintian-overrides ${S}/usr/share/lintian/overrides/apparmor-utils
+    #debian/apparmor-easyprof.install
+    install -D ${S}/debian/apparmor-easyprof.lintian-overrides ${S}/usr/share/lintian/overrides/apparmor-easyprof
+}
+
+#Building ptest on arm fails.
+do_compile_ptest_aarch64 () {
+  :
+}
+
+do_compile_ptest_arm () {
+  :
+}
+
+do_compile_ptest () {
+    oe_runmake -C ${B}/tests/regression/apparmor
+    oe_runmake -C ${B}/parser/tst
+    oe_runmake -C ${B}/libraries/libapparmor
+}
+
+#Building ptest on arm fails.
+do_install_ptest_aarch64 () {
+  :
+}
+
+do_install_ptest_arm() {
+  :
+}
+
+do_install_ptest () {
+    t=${D}/${PTEST_PATH}/testsuite
+    install -d ${t}
+    install -d ${t}/tests/regression/apparmor
+    cp -rf ${B}/tests/regression/apparmor ${t}/tests/regression
+
+    install -d ${t}/parser/tst
+    cp -rf ${B}/parser/tst ${t}/parser
+    cp ${B}/parser/apparmor_parser ${t}/parser
+    cp ${B}/parser/frob_slack_rc ${t}/parser
+
+    install -d ${t}/libraries/libapparmor
+    cp -rf ${B}/libraries/libapparmor ${t}/libraries
+
+    install -d ${t}/common
+    cp -rf ${B}/common ${t}
+
+    install -d ${t}/binutils
+    cp -rf ${B}/binutils ${t}
+}
+
+INITSCRIPT_PACKAGES = "${PN}"
+INITSCRIPT_NAME = "apparmor"
+INITSCRIPT_PARAMS = "start 16 2 3 4 5 . stop 35 0 1 6 ."
+
+SYSTEMD_PACKAGES = "${PN}"
+SYSTEMD_SERVICE_${PN} = "apparmor.service"
+SYSTEMD_AUTO_ENABLE = "disable"
+
+PACKAGES += "mod-${PN}"
+
+FILES_${PN} += "/lib/apparmor/ ${sysconfdir}/apparmor ${libdir}/python3/dist-packages /usr/share/" 
```

Comment:
${datadir} would be nice instead of /usr/share.
FYI: That value is defined in [meta/conf/bitbake.conf](https://git.yoctoproject.org/poky/tree/meta/conf/bitbake.conf?h=warrior)

Reply:
Fix /usr/share to ${datadir}